### PR TITLE
feat: bump youtube patches to 17.32.35

### PR DESCRIPTION
**Pull request by inotia00**
I'm leaving a PR for the first time
If there are any mistakes in the code or if there are any parts that need correction, please give me feedback
YouTube versions I tested: v17.17.34, v17.25.34, v17.29.34, v17.32.35

### CODE REVIEW

File: src/main/kotlin/app/revanced/patches/youtube/layout/shorts/button/fingerprints/PivotBarButtonTabEnumFingerprint.kt [hunk 1]
```diff
@@ -22,9 +22,6 @@ object PivotBarButtonTabEnumFingerprint : MethodFingerprint(
     AccessFlags.PUBLIC or AccessFlags.FINAL,
     listOf("Z"),
     listOf(
-        Opcode.CHECK_CAST,
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/youtube/layout/shorts/button/fingerprints/PivotBarButtonTabEnumFingerprint.kt:25]: Is the pattern still gonna work for the versions listed in `ShortsButtonCompatibility`? If you are not sure, remove all versions you have not tested this pattern.

File: src/main/kotlin/app/revanced/patches/youtube/layout/shorts/button/fingerprints/PivotBarButtonsViewFingerprint.kt [hunk 2]
```diff
@@ -20,11 +20,6 @@ object PivotBarButtonsViewFingerprint : MethodFingerprint(
     AccessFlags.PUBLIC or AccessFlags.FINAL,
     listOf("Z"),
     listOf(
-        Opcode.INVOKE_VIRTUAL_RANGE,
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/youtube/layout/shorts/button/fingerprints/PivotBarButtonsViewFingerprint.kt:23]: Is this also compatible? Same procedure as the other fingerprint.

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

File: src/main/kotlin/app/revanced/patches/youtube/layout/shorts/button/fingerprints/PivotBarButtonTabEnumFingerprint.kt (hunk 1 above)
Comment (inotia00) [reply src/main/kotlin/app/revanced/patches/youtube/layout/shorts/button/fingerprints/PivotBarButtonTabEnumFingerprint.kt:25]: > Is the pattern still gonna work for the versions listed in `ShortsButtonCompatibility`? If you are not sure, remove all versions you have not tested this pattern.

As per your advice I've tested it on all versions marked in ShortsButtonCompatibility
: 17.14.35, 17.17.34, 17.19.36, 17.20.37, 17.22.36, 17.23.35, 17.23.36, 17.24.34, 17.24.35, 17.25.34, 17.26.35, 17.27.39, 17.28 .34, 17.29.34, 17.32.35
The 'hide-shorts-button' and 'disable-create-button' patches worked fine in all versions marked in ShortsButtonCompatibility.

Similarly, I have also tested all versions marked in WideSearchbarCompatibility.
: 17.25.34, 17.26.35, 17.27.39, 17.28.34, 17.29.34, 17.32.35
The 'enable-wide-searchbar' patch worked fine in all versions indicated in WideSearchbarCompatibility.

**Comment by oSumAtrIX** [comment]
Thanks

**Comment by oSumAtrIX** [comment]
Thanks

**Pull request merged**